tools/deployment/package/docker-compose-all.yaml

```diff
@@ -486,3 +486,30 @@ services:
         "-f",
         "http://mcp_server:8000/health"
       ]
+
+  api-server:
```

Comment:
let's also update `user-docs/guides-multi-host` and `dev-docs/design-deployment-orchestration`